Description: remove getrandom dependency
 getrandom is used nowhere in the code, so remove it.
 getrandom in debian doesn't provide the "js" feature.
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -91,10 +91,6 @@
 [dev-dependencies.bumpalo]
 version = "3.14.0"
 
-[dev-dependencies.getrandom]
-version = "^0.2"
-features = ["js"]
-
 [dev-dependencies.libc]
 version = "0.2.163"
 
